modules/reactor/src/meshgenerators/CoreMeshGenerator.C

```diff
@@ -186,21 +191,33 @@ CoreMeshGenerator::CoreMeshGenerator(const InputParameters & parameters)
     if (make_empty)
     {
       {
-        auto params = _app.getFactory().getValidParams(
-            "HexagonConcentricCircleAdaptiveBoundaryMeshGenerator");
-
-        params.set<Real>("hexagon_size") = getReactorParam<Real>("assembly_pitch") / 2.0;
-        params.set<std::vector<unsigned int>>("num_sectors_per_side") =
-            std::vector<unsigned int>(6, 2);
-        params.set<std::vector<unsigned int>>("sides_to_adapt") = std::vector<unsigned int>{0};
-        params.set<std::vector<MeshGeneratorName>>("inputs") =
-            std::vector<MeshGeneratorName>{_inputs[0]};
-        params.set<std::vector<subdomain_id_type>>("background_block_ids") =
-            std::vector<subdomain_id_type>{UINT16_MAX - 1};
-
-        addMeshSubgenerator("HexagonConcentricCircleAdaptiveBoundaryMeshGenerator",
-                            std::string(_empty_key),
-                            params);
+        if (assembly_homogenization)
```

Comment:
why not the adaptive boundary for homogenized mesh?

Reply:
Adaptive boundary is not currently compatible with SimpleHexagonGenerator since it assumes that there are an even number of sectors on each side of the hexagon, which is not the case for SimpleHexagonGenerator

Reply:
ok let s just add a node to the CoreMeshGen documentation that this is not supported

Reply:
Done, let me know if the wording should be updated